src/Kdyby/Translation/PrefixedTranslator.php

```diff
@@ -49,6 +49,10 @@ public function __construct($prefix, ITranslator $translator)
 
 	public function translate($message, $count = NULL, $parameters = array(), $domain = NULL, $locale = NULL)
 	{
+		if ($message instanceof Phrase) {
+			return $message->translate($this);
```

Comment:
Oh, I totally overlook that the `Phrase`'s properties are actually public, that's why I made this change and not the one you've suggested. Will update.